src/status_im/contexts/communities/actions/request_to_join/view.cljs

```diff
@@ -5,19 +5,22 @@
     [quo.theme]
     [react-native.core :as rn]
     [react-native.gesture :as gesture]
-    [status-im.common.password-authentication.view :as password-authentication]
     [status-im.contexts.communities.actions.community-rules-list.view :as community-rules]
     [status-im.contexts.communities.actions.request-to-join.style :as style]
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
 (defn join-community-and-navigate-back
-  [id]
-  (rf/dispatch [:password-authentication/show
-                {:content (fn [] [password-authentication/view])}
-                {:label    (i18n/label :t/join-open-community)
-                 :on-press #(rf/dispatch [:communities/request-to-join
-                                          {:community-id id :password %}])}])
+  [id theme]
+  (rf/dispatch
+   [:standard-auth/authorize
+    {:auth-button-label (i18n/label :t/request-to-join)
+     :theme             theme
```

Comment:
Why this one has `:theme`? Or should we also add this key to above cases?

Reply:
Will double check, I remember that it might not be needed.